tests/conftest.py

```diff
@@ -21,12 +21,18 @@
 
 from tests import path_of_for_test
 
+import browse.services.documents as documents
+from browse.services.listing import db_listing
+
 DEFAULT_DB = "sqlite:///../tests/data/browse.db"
 TESTING_LATEXML_DB = 'sqlite:///../tests/data/latexmldb.db'
 
 
 TESTING_CONFIG = {
     "SQLALCHEMY_BINDS": {"latexml": TESTING_LATEXML_DB},
+    "SQLALCHEMY_DATABASE_URI" : DEFAULT_DB,
+    'DOCUMENT_LISTING_SERVICE': db_listing,
```

Comment:
i dont think this is fully/ properly implemented. I believe tests either use the fake listing service or the file listing service with the listing file path pointing to local test data